sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-by-filter/README.md

```diff
@@ -60,28 +60,28 @@ As a first step you'll need to:
    - In the **Supported account types** section, select **Accounts in any organizational directory**.
    - Choose **Single-page application(SPA)** as application type.
    - Add `http://localhost:8080` as the `Reply URL` under Redirect URI.
-   - Select **Register** to create the application. ![create the application](docs/application-register.png "create the application")
-   - After creating the application, on the application **Overview** page, click the **Redirect URIs** to edit, select the **Access tokens** and **ID tokens**, and click **Save**. ![add tokens](docs/add_tokens.png "add tokens")
+   - Select **Register** to create the application. ![register_the application](docs/application-registration-1.png "register the application")![register_the application](docs/application-registration-2.png "register the application")
+   - After creating the application, on the application **Overview** page, click the **Redirect URIs** to edit, select the **Access tokens** and **ID tokens**, and click **Save**. ![accept tokens](docs/tokens-to-accept.png "accept tokens")
 1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the application.properties file for this project.
 1. On selecting your application from the the registered applcations you can see **Certificates & secrets** in left navigation pane, go to that page and in the **Client secrets** section, choose **New client secret**:
 
    - Type a key description (of instance `app secret`),
-   - Select a key duration of either **In 1 year**, **In 2 years**, or **Never Expires**. ![create the secret](docs/create_secret.png "create the secret")
-   - When you press the **Add** button, the key value will be displayed, copy, and save the value in a safe location. ![secret value](docs/secret_value.png "secret value")
+   - Select a key duration of either **In 1 year**, **In 2 years**, or **Never Expires**. ![create the secret](docs/create-secret.png "create the secret")
```

Comment:
Delete content about setting expire time. Because it's not required for our sample. User can decide the expire time by himself.